src/Analysis/Ast/Impl/Modules/PythonModule.cs

```diff
@@ -243,6 +243,24 @@ internal PythonModule(ModuleCreationOptions creationOptions, IServiceContainer s
             }
         }
 
+        /// <summary>
+        /// Returns the string line corresponding to the given location
+        /// </summary>
+        /// <param name="line">The line number</param>
+        public string GetLine(int line) {
+            StringBuilder sb = new StringBuilder();
+            SourceLocation source = new SourceLocation(line, 1);
+            string content = Content;
+            var index = this.GetAst().LocationToIndex(source);
+
+            if (!content.IsNullOrEmpty()) {
```

Comment:
This is probably not the most efficient way to do this; it's likely easier to loop and scan for the end of line index and then slice out the part you want with Substring.

Reply:
Changed.